site/content/docs/09-building.md

```diff
@@ -14,6 +14,8 @@ The output is a Node app that you can run from the project root:
 node __sapper__/build
 ```
 
+If you're deploying your app outside your machine, make sure to satisfy your `server.js` by installing all your dependencies from `package.json` and to also include your `static` folder.
```

Comment:
```suggestion
If you're deploying your app outside your machine, make sure that you include production dependencies (required by) `server.js` by installing all dependencies from `package.json`:

 ```
   npm --production
 ```
 
   as well as including your `static` directory and it's contents.
```

not sure if we should also include `npm prune`, but I figure that we're not an npm tutorial.